src/core/meshcop/tcat_agent.cpp

```diff
@@ -540,6 +547,97 @@ Error TcatAgent::HandlePing(const Message &aIncomingMessage,
     return error;
 }
 
+Error TcatAgent::HandleGetNetworkName(Message &aOutgoingMessage)
+{
+    Error       error  = kErrorNone;
+    const char *name   = otThreadGetNetworkName(&GetInstance());
+    uint16_t    length = static_cast<uint16_t>(strlen(name));
+    ot::Tlv     tlv;
+
+    VerifyOrExit(Get<ActiveDatasetManager>().IsCommissioned(), error = kErrorInvalidState);
+    VerifyOrExit(length > 0, error = kErrorInvalidState);
```

Comment:
In this case what should we return? Response with status of size 0?

Reply:
Yes, I think so. 
Basically we append it as an empty TLV (TLV with length zero). When parsing such a TLV (with string value), it would be read as an empty string.